daemon/src/lib.rs

```diff
@@ -421,17 +421,25 @@ where
             .context("Price feed not available")?
             .context("No quote available")?;
 
-        if latest_quote.is_older_than(QUOTE_INTERVAL_MINUTES.minutes()) {
+        let current_price = Price::new(latest_quote.for_taker())?;
+        let quote_timestamp = latest_quote
+            .timestamp
+            .format(&time::format_description::well_known::Rfc3339)
+            .context("Failed to format timestamp")?;
+
+        tracing::debug!(%order_id, %current_price, %quote_timestamp, "Proposing settlement of contract");
+
+        if latest_quote.is_older_than(QUOTE_INTERVAL_MINUTES.minutes() * 2) {
             anyhow::bail!(
                 "Latest quote is older than {} minutes. Refusing to settle with old price.",
-                QUOTE_INTERVAL_MINUTES
+                QUOTE_INTERVAL_MINUTES * 2
```

Comment:
nit: this sort of duplication is prone to rot (if we ever decide to change the solution to change the multiplier, there's high likelyhood that the log won't get updated and will be actively misleading.

I'd suggest extracting `let quote_threshold = QUOTE_INTERVAL_MINUTES * 2;` and then using it in both places

Reply:
Agreed. Will send a follow-up but merge this now so autotaker can close positions again!